### ISSUE TITLE
[Test] Is container working as expected?

### ISSUE THREAD

**Issue by zaydek** [zaydek/duomo#90]

**Comment by zaydek** [comment]
The problem is were coupling the responsive class names when it should be automatic:

**Current output:**

```css
.container {
  width: 100%;
}

@media (min-width: 512px) {
  .xs\:container {
    max-width: 512px;
  }
}
@media (min-width: 640px) {
  .sm\:container {
    max-width: 640px;
  }
}
@media (min-width: 768px) {
  .md\:container {
    max-width: 768px;
  }
}
@media (min-width: 1024px) {
  .lg\:container {
    max-width: 1024px;
  }
}
@media (min-width: 1280px) {
  .xl\:container {
    max-width: 1280px;
  }
}
```

**Desired output:**

```css
.container {
  width: 100%;
}

@media (min-width: 512px) {
  .container {
    max-width: 512px;
  }
}
@media (min-width: 640px) {
  .container {
    max-width: 640px;
  }
}
@media (min-width: 768px) {
  .container {
    max-width: 768px;
  }
}
@media (min-width: 1024px) {
  .container {
    max-width: 1024px;
  }
}
@media (min-width: 1280px) {
  .container {
    max-width: 1280px;
  }
}
```

We may want to also support `2xl` out of the box?

**Comment by zaydek** [comment]
Solved by v0.6.19.
